src/lib.rs

```diff
@@ -12,6 +12,8 @@ extern crate alloc;
 
 extern crate packed_struct;
 
+extern crate crc_any;
```

Comment:
I prefer the `use` style imports, anyhow this surely better fits the rest of the code the way it is.

Reply:
i just tried to make it work, not so proficient with rust to have opinion yet